shared/middleware/helpers/null_pub_sub_api.rb

```diff
@@ -0,0 +1,11 @@
+# Interface to fake interactions with a PubSub system, like Pusher
+# Should respond to same messages as PusherApi
+
+class NullPubSubApi
```

Comment:
I would be tempted to call this "FakePusherAPI", if I understand it correctly.  It appears to be a stubbed out implementation of the Pusher API rather than the generic pubsub API.